cms/envs/acceptance.py

```diff
@@ -127,3 +127,7 @@ def seed():
 # Generate a random UUID so that different runs of acceptance tests don't break each other
 import uuid
 SECRET_KEY = uuid.uuid4().hex
+
+############################### PIPELINE #######################################
+
+PIPELINE_ENABLED = False
```

Comment:
Never mind, just saw your comment below.  Since we're not writing new lettuce tests, I suppose we can use whatever settings allow the current ones to pass :)